app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -68,7 +68,7 @@ <h1>Reader Study {{ object.title }}</h1>
         <div class="h3">
```

Comment:
Actually, this looks weird in an h3 block. Maybe make this change:

```suggestion
        <div class="alert alert-info" role="alert">
```

Reply:
Yes, that looks a lot better.